fabtests/pytest/common.py

```diff
@@ -292,6 +292,9 @@ def prepare_base_command(self, command_type, executable,
                 pytest.skip("no cuda device")
             elif host_memory_type == "neuron" and not has_neuron(host_ip):
                 pytest.skip("no neuron device")
+            elif (client_memory_type == server_memory_type == "neuron") and (
+                    self._cmdline_args.server_id == self._cmdline_args.client_id):
+                pytest.skip("Neuron to Neuron tests require 2 nodes")
```

Comment:
only "neuron to neuron" has this limitation?

Reply:
Right. The failure mode is that we cannot run 2 processes with `-D neuron` on the same node. The 1st process will occupy all Neuron cores, and the 2nd process will fail because there is no Neuron cores available.